### DISCUSSION TITLE
.wiff file containing 2 files

### DISCUSSION THREAD

**Discussion by brvpuyve** [vdemichev/DiaNN#157] [Q&A]
Hi Vadim, 

I have a Scanning SWATH run but apparently the .wiff file contains two different files (the first one is empty) and DIA-NN is not able to find the appropriate .scan file although it is in the folder. Any suggestions on how this could be resolved? 

Best,
Bart

**Comment by vdemichev** [comment]
Hi Bart,

Please ask Sciex guys why the file contains two files. DIA-NN works fine with our sswath runs.

Best,
Vadim

**Reply by brvpuyve** [reply]
Hi Vadim,

Perfect, will do!
